Palmr Script fails to install due to pnpm prisma error

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Palmr

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/palmr.sh)"

### ⚙️ What settings are you using?

- [x] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 📈 Which Proxmox version are you on?

pve-manager/8.4.1/2a5fa54a8503f96d (running kernel: 6.8.12-11-pve)

### 📝 Provide a clear and concise description of the issue.

Palmr is not installing due to an issue with the pnpm module Prisma: `[ERROR] in line 35: exit code 0: while executing command $STD pnpm dlx prisma generate`

### 🔄 Steps to reproduce the issue.

Use the script directly in the Proxmox Shell: `bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/palmr.sh)"`

### ❌ Paste the full error output (if available).

All good until here:

```
✔️   Installed Node.js modules: pnpm@10.6.0
  ⏳   Configuring palmr backendLockfile is up to date, resolution step is skipped
Packages: +448
+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

   ╭───────────────────────────────────────────────────────────────────╮
   │                                                                   │
   │                Update available! 10.6.0 → 10.15.1.                │
   │   Changelog: https://github.com/pnpm/pnpm/releases/tag/v10.15.1   │
   │                 Run "pnpm add -g pnpm" to update.                 │
   │                                                                   │
   ╰───────────────────────────────────────────────────────────────────╯

Downloading prisma@6.11.1: 6.30 MB/6.30 MB, done
Downloading @prisma/client@6.11.1: 9.48 MB/9.48 MB, done
Downloading @img/sharp-libvips-linuxmusl-x64@1.1.0: 7.53 MB/7.53 MB, done
Progress: resolved 448, reused 0, downloaded 448, added 448, done
Downloading @img/sharp-libvips-linux-x64@1.1.0: 7.40 MB/7.40 MB, done

dependencies:
+ @aws-sdk/client-s3 3.842.0
+ @aws-sdk/s3-request-presigner 3.842.0
+ @fastify/cookie 11.0.2
+ @fastify/cors 10.1.0
+ @fastify/jwt 9.1.0
+ @fastify/multipart 9.0.3
+ @fastify/static 8.2.0
+ @fastify/swagger 9.5.1
+ @fastify/swagger-ui 5.2.3
+ @prisma/client 6.11.1
+ @scalar/fastify-api-reference 1.32.1
+ @types/crypto-js 4.2.2
+ bcryptjs 2.4.3
+ crypto-js 4.2.0
+ fastify 5.4.0
+ fastify-type-provider-zod 4.0.2
+ jose 5.10.0
+ node-fetch 3.3.2
+ nodemailer 6.10.1
+ openid-client 6.6.2
+ qrcode 1.5.4
+ sharp 0.34.2
+ speakeasy 2.0.0
+ zod 3.25.74

devDependencies:
+ @eslint/eslintrc 3.3.1
+ @eslint/js 9.30.0
+ @ianvs/prettier-plugin-sort-imports 4.4.2
+ @types/bcryptjs 2.4.6
+ @types/node 22.16.0
+ @types/nodemailer 6.4.17
+ @types/qrcode 1.5.5
+ @types/speakeasy 2.0.10
+ @typescript-eslint/eslint-plugin 8.35.1
+ @typescript-eslint/parser 8.35.1
+ eslint 9.30.0
+ eslint-config-prettier 9.1.0
+ eslint-plugin-prettier 5.5.1
+ prettier 3.6.2
+ prettier-plugin-sort-json 4.1.1
+ prisma 6.11.1
+ ts-node 10.9.2
+ tsx 4.20.3
+ typescript 5.8.3

╭ Warning ───────────────────────────────────────────────────────────────────────────────────╮
│                                                                                            │
│   Ignored build scripts: @prisma/client, @prisma/engines, esbuild, prisma, sharp.          │
│   Run "pnpm approve-builds" to pick which dependencies should be allowed to run scripts.   │
│                                                                                            │
╰────────────────────────────────────────────────────────────────────────────────────────────╯

Done in 7.1s using pnpm v10.6.0
Downloading prisma@6.16.0: 17.56 MB/17.56 MB, done
Packages: +33
+++++++++++++++++++++++++++++++++
Progress: resolved 33, reused 0, downloaded 33, added 33, done
.cache/pnpm/dlx/76f71f30b656c4a16f70759bdebc73d84c20328681685ff2d436a56fe7c975f0/19934375bff-2cbd/node_modules/.pnpm/prisma@6.16.0/node_modules/prisma: Running preinstall .cache/pnpm/dlx/76f71f30b656c4a16f70759bdebc73d84c20328681685ff2d436a56fe7c975f0/19934375bff-2cbd/node_modules/.pnpm/prisma@6.16.0/node_modules/prisma: Running preinstall script, done in 60ms
warn The configuration property `package.json#prisma` is deprecated and will be removed in Prisma 7. Please migrate to a Prisma config file (e.g., `prisma.config.ts`).
For more information, see: https://pris.ly/prisma-config

Environment variables loaded from .env
Prisma schema loaded from prisma/schema.prisma
Error: 
Cannot find module '/opt/palmr/apps/server/node_modules/.pnpm/@prisma+client@6.11.1_prisma@6.11.1_typescript@5.8.3__typescript@5.8.3/node_modules/@prisma/client/runtime/query_engine_bg.sqlite.wasm-base64.js'
Require stack:
- /root/.cache/pnpm/dlx/76f71f30b656c4a16f70759bdebc73d84c20328681685ff2d436a56fe7c975f0/19934375bff-2cbd/node_modules/.pnpm/prisma@6.16.0/node_modules/prisma/build/index.js



[ERROR] in line 35: exit code 0: while executing command $STD pnpm dlx prisma generate

[ERROR] in line 1345: exit code 0: while executing command lxc-attach -n "$CTID" -- bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/install/${var_install}.sh)"
```

### 🖼️ Additional context (optional).

_No response_

--- a/ct/palmr.sh
+++ b/ct/palmr.sh
@@ -42,9 +42,9 @@ function update_script() {
     cd /opt/palmr/apps/server
     mv /opt/palmr.env /opt/palmr/apps/server/.env
     $STD pnpm install
-    $STD pnpm dlx prisma generate
-    $STD pnpm dlx prisma migrate deploy
-    $STD pnpm dlx prisma db push
+    $STD npx prisma generate
+    $STD npx prisma migrate deploy
+    $STD npx prisma db push
     $STD pnpm build
 
     cd /opt/palmr/apps/web

--- a/install/palmr-install.sh
+++ b/install/palmr-install.sh
@@ -32,9 +32,9 @@ sed -e 's/_ENCRYPTION=true/_ENCRYPTION=false/' \
 # CUSTOM_PATH=<path-to-your-bind-mount>" \
   .env.example >./.env
 $STD pnpm install
-$STD pnpm dlx prisma generate
-$STD pnpm dlx prisma migrate deploy
-$STD pnpm dlx prisma db push
+$STD npx prisma generate
+$STD npx prisma migrate deploy
+$STD npx prisma db push
 $STD pnpm db:seed
 $STD pnpm build
 msg_ok "Configured palmr backend"

